src/location/schemas/location.schema.ts

```diff
@@ -0,0 +1,36 @@
+import { HydratedDocument } from 'mongoose';
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+
+export type LocationDocument = HydratedDocument<Location>;
+
+@Schema()
+export class Location {
+  @Prop()
+  _id: string;
+
+  @Prop({ required: true })
+  CountryCode: string;
```

Comment:
should be in camelCase (countryCode)

Reply:
Changed all properties to camelCase, including the headers in csv files